packages/panda-san/src/navbar/navbar.scss

```diff
@@ -0,0 +1,107 @@
+.navbar {
+  align-items: center;
+  background-color: $navbar;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  left: 0;
+  position: absolute;
+  width: 100vw;
+  z-index: 100;
+
+  &__brand {
+    display: flex;
+    flex: 1;
+
+    &__icon {
+      cursor: pointer;
+    }
+  }
+
+  &__burger {
+    display: flex;
+    flex: 1;
+    justify-content: flex-end;
+
+    button {
+      background: none;
+      border: 0;
+    }
+  }
+
+  &__menu {
+    display: none;
+
+    &,
+    ul,
+    ol {
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    &--active {
+      background-color: $navbar-menu;
+      display: flex;
+      flex-direction: column;
+      position: relative;
+      width: 100vw;
+    }
+  }
+
+  &__item {
+    align-items: center;
+    display: flex;
+    height: px-to-rem(36px);
+    justify-content: flex-start;
+    width: 100%;
+
+    &:hover,
+    &:focus {
+      background-color: darken($navbar-menu, 10%);
+    }
+
+    &--link {
+      color: $link;
+      text-decoration: none;
```

Comment:
We should probably have link and button reset mixins. We can think of adding that later.